packages/salesforcedx-vscode-core/src/commands/forceProjectCreate.ts

```diff
@@ -144,10 +143,6 @@ export class SelectProjectTemplate
       new ProjectTemplateItem(
         projectTemplateEnum.empty,
         'force_project_create_empty_template'
-      ),
-      new ProjectTemplateItem(
-        projectTemplateEnum.analytics,
-        'force_project_create_analytics_template'
       )
     ];
 
```

Comment:
do we need to modify the build method to use the `--standard` flag for the template instead of `data.projectTemplate` again?

Reply:
It's okay as is. We're just passing `standard` or `empty` to the `--template` flag depending on the selection